spec/features/application/when_user_is_logged_in_navbar_shows_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "rails_helper"
+
+feature "when user is logged in navbar shows" do
```

Comment:
Same here. These specs should be view specs and they will run faster

Reply:
Done by @elle sha: 2985975